coalib/bearlib/languages/definitions/Tcl.py

```diff
@@ -0,0 +1,12 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Tcl:
+    aliases = 'Tcl/Tk',
+    versions = 8.0, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6
```

Comment:
as with m4, I dont feel we're in a good position to define language versions at this stage.
In this case, Tcl the program and Tcl the language were quite closely tied together, but we dont know which program releases included language syntax changes.

Reply:
Right, I better remove versions